src/apicheck.ts

```diff
@@ -13,6 +13,7 @@ let apiContainer = {
   optional: [
     "performance",
     "performance.getEntriesByType",
+    "window.addEventListener"
   ]
```

Comment:
What about older IE browser (pre IE9) that don't have this? Should you use the equivalent call (EventTarget.attachEvent ) using feature detection or should we just not target those old browsers?